CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Change Log
 
+## [v1.117.0] - 2024-06-04
+
+- #697 - @asaha2 - Add support for lb internal network type
```

Comment:
```suggestion
- #696 - @llDrLove - Support specifying control plane firewall rules when creating or updating DOKS clusters
- #697 - @asaha2 - Add support for lb internal network type
- #695 - @ElanHasson - APPS-8732 - Update documentation on App Platform OpenSearch endpoint structure.
- #692 - @ElanHasson - APPS-8732 - Add OpenSearch as a Log Destination for App Platform.
```

Reply:
Ah yes, sorry I missed backtracking the commits.